app/lib/actions/katello/host/erratum/install.rb

```diff
@@ -39,6 +40,11 @@ def resource_locks
           def presenter
             Helpers::Presenter::Delegated.new(self, planned_actions(Pulp::Consumer::ContentInstall))
           end
+
+          def finalize
+            host = ::Host.find_by(:id => input[:host_id])
+            host.update(audit_comment: _("Installation of errata requested: %{errata}") % {errata: input[:errata].join(", ")})
```

Comment:
where is `input[:errata]` being defined? The `plan_self` only has `:host_id` in it

Reply:
I guess, the input[:errata] is send the same way as it is send in humanized_input. 
It works. 

![image](https://user-images.githubusercontent.com/25485845/48134205-9b68a400-e299-11e8-8444-2b447655e1e3.png)